ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/attributes-rearrange/dropdown.attributes-rearrange.less

```diff
@@ -1,8 +1,6 @@
 @{customElementNamespace}dropdown.is-attributesRearrange {
   cursor: pointer;
-  display: block;
-
-  .dropdown-text {
-    display: inline-block;
-  }
+  display: inline-block;
+  position: relative;
+  margin-left: 10px;
```

Comment:
Is it possible to use scalable-units here and achieve the same effect?

Reply:
good call 👍